atoti-query-analyser/src/components/Details/Details.tsx

```diff
@@ -93,18 +93,28 @@ function PlainView({ value }: { value: unknown }) {
  * @param attributes.title - List title
  * */
 function ListView({ title, list }: { title: string; list: unknown[] }) {
+  const MAX_ELEMENTS = 10;
   return (
     <li>
       {title}:
       {list.length === 0 ? (
         <span className="nullish-content"> (no items)</span>
       ) : (
         <ul>
-          {list.map((item) => (
-            <li key={`${item}`}>
-              <PlainView value={item} />
+          {list
+            .filter((_, idx) => idx < MAX_ELEMENTS)
+            .map((item) => (
+              <li key={`__element__${item}`}>
+                <PlainView value={item} />
+              </li>
+            ))}
+          {list.length > MAX_ELEMENTS && (
+            <li key="__ETC__">
+              <span className="nullish-content">
+                And {list.length - MAX_ELEMENTS} more items
+              </span>
             </li>
-          ))}
+          )}
```

Comment:
Why not, but this means that it is not possible from the UI to access the other elements.
I suppose that one can always filters its list then passing many distincts plans.
However, this would block users with multiple MDX passes.
We could at least increase this to 15.

Reply:
You can use this CSS property https://developer.mozilla.org/fr/docs/Web/CSS/overflow-y to allow scrolling when you click on some "more" button (to add)

Reply:
Still this point to do before mergin.